### DISCUSSION TITLE
Recalculate formulas in cells during parsing

### DISCUSSION THREAD

**Discussion by cronosnn** [ozlerhakan/poiji#269] [Q&A]
Hi, do you have any possibility to recalculate formulas in the cells during parsing or before parsing?
Or I should use another lib to recalculate before parsing excel?
Thanks.

**Comment by ozlerhakan** [comment]
Hi @cronosnn , on Poiji side, it is only able to read data from the excel file. We can look into possibilities if there is something if Poiji could handle through the logic.
